ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragment/v1/fragment/clientlibs/site/css/fragmentview.css

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+.cmp-adaptiveform-fragment {
+
+}
+
+.cmp-adaptiveform-fragment__widget {
+
+}
+textarea.cmp-adaptiveform-fragment__widget {
+  
+}
+
+.cmp-adaptiveform-fragment__label {
+
+}
+
+.cmp-adaptiveform-fragment__longdescription {
+}
+
+.cmp-adaptiveform-fragment__shortdescription {
+}
+
+.cmp-adaptiveform-fragment__questionmark {
+
+}
+
+.cmp-adaptiveform-fragment__placeholderContainer {
+    border-opacity: .1;
+    border-width: 1px;
+    padding: 15px;
+    font-family: Adobe Clean;
+    font-weight: normal;
+    font-size: 13px;
+    text-align: left;
+    color: #969696;
+    border-color: rgba(0,0,0,0.15);
+    border-radius: 4px;
+    background: linear-gradient(180deg,rgba(0,0,0,0.05),rgba(255,255,255,0.05),rgba(0,0,0,0.05)),
+                linear-gradient(90deg,rgba(0,0,0,0.05),rgba(255,255,255,0.05),rgba(0,0,0,0.05));
+}
+
+.cmp-adaptiveform-fragment__placeholderContainer hr {
+    width: auto;
+    height: 1px;
+    background-color: #000;
+    opacity: .25;
+    margin: 15px 0;
+    border-style: solid;
+    border-color: rgba(0,0,0,0.15);
+}
+
+.cmp-adaptiveform-fragment__fragmentTitle {
```

Comment:
The element in BEM could just be title, can you rename this to `cmp-adaptiveform-fragment__title`

Reply:
Again this is title of AF Fragment referred by the component, `title` is generally 'jcr:title` of component but this is a different one.